src/test/GradleTestDevModeActionsWSProjectNameInit.ts

```diff
@@ -0,0 +1,20 @@
+import {  WebDriver, VSBrowser } from 'vscode-extension-tester';
+import * as utils from './utils/testUtils';
+
+describe('Open Gradle Project - Project name with space', () => {
+
+    let driver: WebDriver;  
+
+    before(() => {    
+        utils.renameProject();  
```

Comment:
Please add comments, that will be helpful to understand about the newly added methods.

Reply:
Comments added for new methods. Thanks.